recipes/net/iproute2.yaml

```diff
@@ -0,0 +1,33 @@
+inherit: [cpackage, make, install]
+
+metaEnvironment:
+    PKG_VERSION: "6.13.0"
+    PKG_LICENSE: "GPL-2.0+"
+
+checkoutSCM:
+    scm: url
+    url: https://mirrors.edge.kernel.org/pub/linux/utils/net/iproute2/iproute2-${PKG_VERSION}.tar.xz
+    digestSHA256: "a43aa43338d882b44d01e549f3f105a92ae9feea32a82fae45a88e7a49302819"
+    stripComponents: 1
+
+buildVars: [CC, AR]
+buildTools: [host-toolchain, bison, flex]
+buildScript: |
+    cp -a $1/ build/
+
+    pushd build
+    ./configure
+
+    makeParallel \
+        CC="$CC" \
+        HOSTCC="gcc" \
```

Comment:
Buildroot also passes `SHARED_LIBS=`. Depending on the target environment, the basement recipes prefer static (host environment) or shared (target environment) libraries. Something like the following should do that:

```
SHARED_LIBS=$([[ $(cpackageLibraryType) == static ]] && echo n || echo y)
```